docs/core/porting/snippets/upgrade-assistant-wcf-framework/CalculatorSample.Upgraded/CalculatorService/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,29 @@
+﻿//  Copyright (c) Microsoft Corporation.  All Rights Reserved.
+
+#region Using directives
+
+using System;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Security.Permissions;
+
+#endregion
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: ComVisible(false)]
+[assembly: CLSCompliant(true)]
+[assembly: SecurityPermissionAttribute(SecurityAction.RequestMinimum)]
+[assembly: AssemblyTitle("service")]
+[assembly: AssemblyDescription("SelfHost")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("Microsoft")]
+[assembly: AssemblyProduct("service")]
+[assembly: AssemblyCopyright("Copyright @ Microsoft 2004")]
+[assembly: AssemblyTrademark("")]
+[assembly: AssemblyCulture("")]
+
+
```

Comment:
```suggestion
```

Reply:
Hi! I updated the documentation to include the major changes after upgrade, and removed the Upgraded folder of the sample.